src/lib/forms/core/VocabularyField.js

```diff
@@ -44,13 +44,24 @@ export class VocabularyField extends React.Component {
     text: hit.metadata.text,
   });
 
+  moveItem = (from, to, data) => {
+    const f = data.splice(from, 1)[0];
+    data.splice(to, 0, f);
+  };
+
   fetchVocabularies = async (serializer) => {
-    const { label } = this.props;
+    const { label, orderBy } = this.props;
     this.cancellableFetchVocabularies = withCancel(this.query());
     try {
       const response = await this.cancellableFetchVocabularies.promise;
       const options = response.data.hits.map((hit) => serializer(hit));
-
+      if (orderBy) {
```

Comment:
I understand why you have implemented like this, but hardcoding `Creative Commons` in the cmp below it means hardcoding CERN CDS behaviour in InvenioILS, and we don't want to do that.

I see 2 options her:
1. manipulate the vocab in the CDS-ILS backend and changed the order of each item, putting `CC` as first
2. introduce an `order` field in the vocab, and then put there an integer, e.g order: 0, order: 1... then, set `CC` as order: 0 and have the Elasticsearch query to `sort=order` and `order:asc` (the `order` field might be a better name eventually).

It would be nice to have the opinion also of @KonstantinaStoikou and @kprzerwa 
I think that we should have a custom manipulation of this vocab, when pushing it to Elasticsearch, and put as first this

Reply:
for short term solution better would be option 1)
but since it is not the single case that we need a specific order in the dictionary, normally I would be in favour of option 2, however because we are supposed to integrate invenio-vocabularies later, the temporary solution seems more convenient for now.
Maybe also opening an issue (or RFC) in invenio-vocabularies would be helpful for the future integration